cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -254,6 +269,23 @@ components:
                 ancestors:
                   FBbt:
                     - FBbt:00007002
+          - # If organism is c. elegans
+            rule: "organism_ontology_term_id == 'NCBITaxon:6239'"
+            error_message_suffix: >-
+              When 'organism_ontology_term_id' is 'NCBITaxon:6239' (Caenorhabditis elegans),
+              'organism_tissue_ontology_term_id' MUST be the most accurate descendant
+              of WBbt:0005766 for Anatomy
+            type: curie
+            curie_constraints:
+              ontologies:
+                - WBbt
+              allowed:
+                ancestors:
+                  WBbt:
+                    - WBbt:0005766
+              forbidden:
+                terms:
+                  - WBbt:0005766
```

Comment:
just curious, do you need to explicitly forbid `WBbt:0005766` if you only allow descendant terms of `WBbt:0005766` on lines 282-285? i had thought the `allowed` block would implicitly forbid the `WBbt:0005766` term, but now i'm not sure and it is different from how the rule for `organism_cell_type_ontology_term_id` is written

Reply:
good catch, I didn't need to add that! For some reason I thought that's what we did, but I was misremembering. fixing!